docs/contributor/04-10-workflows.md

```diff
@@ -38,10 +38,14 @@ This [workflow](/.github/workflows/run-govulncheck.yaml) runs the Govulncheck.
 
 This [workflow](/.github/workflows/auto-merge.yaml) enables the auto-merge functionality on a PR that is not a draft.
 
-## All Cheks Passed Workflow
+## All Checks Passed Workflow
 
 This [workflow](/.github/workflows/pr-checks.yaml) checks if all jobs, except those excluded in the workflow configuration, have passed.
 
+## E2E SAP BTP Manager Secret Customization Test Workflow
```

Comment:
We don't use the name "SAP BTP Manager" to refer to [BTP Manager](https://github.com/kyma-project/btp-manager/blob/main/README.md), so can we drop "SAP" from the test name? Or, let's use the technical name: `sap-btp-manager` Secret to avoid ambiguity. WDYT?

Reply:
I removed "SAP" prefix from names.